hw/top_earlgrey/data/ip/chip_rv_timer_testplan.hjson

```diff
@@ -16,5 +16,56 @@
       stage: V2
       tests: ["chip_sw_rv_timer_irq"]
     }
+    {
+      name: chip_sw_timer
+      desc: '''Verify the timeout interrupt assertion.
+
+            - Configure RV_TIMER to generate an interrupt after a set timeout.
+            - Issue a WFI to wait for the interrupt to trigger.
+            - Service the interrupt when it triggers; verify that it came from rv_timer.
+            - Verify that the interrupt triggered only after the timeout elapsed.
+            '''
+      stage: V2
+      si_stage: SV2
+      features: ["RV_TIMER.ENABLE", "RV_TIMER.CONFIG", "RV_TIMER.COMPARE", "RV_TIMER.INTERRUPT",
+                 "RV_TIMER.COUNTER"]
+      tests: ["chip_sw_rv_timer_irq"]
+    }
+    {
+      name: tick_configuration
+      desc: '''Verify that the timer can be configured to generate a system tick.
+
+            - Configure the timer to generate a tick of `T` microseconds long.
+            - Enable the timer.
+            - Start a busy loop of 1 second based on the `mcycleh` CSR.
+            - Read the number of ticks `n`.
+            - Verify that `n * T ~= 1 second` within 5% of tolerance.
+            - Reapeat for T in :
+                [1, 10, 100 and 1000]
+            '''
+      stage: V3
+      si_stage: SV2
+      features: ["RV_TIMER.ENABLE", "RV_TIMER.CONFIG", "RV_TIMER.COMPARE"]
+      tests: [""]
```

Comment:
I think you want `[]` here: it's an empty list of tests (because this hasn't been implemented yet), rather than a single test with a zero-length name :-)

Similarly for `riscv_isa_csrs` below.